packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -752,8 +756,15 @@ class DDSMastheadL1 extends StableSelectorMixin(LitElement) {
         `
       : html` ${heading.title} `;
 
+    let noLink: boolean = false;
+
+    if (!heading.url) {
+      noLink = true;
+    }
+
     const headingClasses = classMap({
       [`${prefix}--masthead__l1-dropdown-heading`]: true,
+      [`no-link`]: noLink,
```

Comment:
As @kennylam mentioned, let's BEM-ify this class name.

For the sake of learning here, though:

Readability boost: we could reduce this all to a single line here.

```
const headingClasses = classMap({
      [`${prefix}--masthead__l1-dropdown-heading`]: true,
      "no-link": Boolean(heading.url) === false,
});
```
The `[\`...\`]` version is only necessary if we want to do the string interpolation like in the line above with `${prefix}` (which we will when we make a BEM version)

We can also do simple JS comparisons on the value side of the key/value pair, making the `noLink` variable and it's logic above unnecessary.